From 54594141b22474cfc1a0dc3f57258a60706e54c0 Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fabian@ritter-vogt.de>
Date: Wed, 27 Dec 2017 18:54:36 +0100
Subject: [PATCH] Replace autologin configuration with a note to use YaST
 instead

sddm looks at sysconfig for autologin and ignores other configuration files.
---
 src/ui/advanceconfig.ui | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

Index: sddm-kcm-5.12.5/src/ui/advanceconfig.ui
===================================================================
--- sddm-kcm-5.12.5.orig/src/ui/advanceconfig.ui
+++ sddm-kcm-5.12.5/src/ui/advanceconfig.ui
@@ -45,15 +45,19 @@
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
-      <item row="0" column="0">
+      <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
-         <string>User:</string>
+         <string>The user for autologin needs to be configured using YaST or by setting DISPLAYMANAGER_AUTOLOGIN in /etc/sysconfig/displaymanager.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
-       <widget class="QComboBox" name="userList"/>
+       <widget class="QComboBox" name="userList">
+        <property name="visible">
+         <bool>false</bool>
+        </property>
+       </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
